### PR TITLE
sample python new scripts

### PR DESCRIPTION
**Pull request by pmandler**
Basic scripts:
freeze-submissions.py: Make submissions non-editable / non-deletable 
get-submissions.py: Output submission info for assigning reveiwers 
create-invitations.py: create subgroups and reviewer invitations

They all need to be run in the same directory as config.py for the conference.  They do not use utils.py.

### CODE REVIEW

**Comment by pmandler** [comment]
Tested in my local copy of MLAV. There shouldn't be anything specific to that workshop.

File: samples/python/create-invitations.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    client = Client(baseurl=args.baseurl)
```
Comment (mspector) [inline samples/python/create-invitations.py]: in all these scripts, you can replace this if/else block with just a single line:

client = Client(baseurl=args.baseurl, username=args.username, password=args.password)

the Client object will search the environment variables for the username and password if they are None

**Comment by pmandler** [comment]
Yes. And have the option to tweak things for each conference.  
If you think it’s better I could add an argument to get the conference directory.

pam

**Comment by mspector** [comment]
I think a conference argument is a good idea (since it doesn't prevent people from copy/pasting it if they want)

**Comment by melisabok** [comment]
We should be careful the program chairs can change data of other conferences, have we tested that?

**Comment by pmandler** [comment]
I’m testing out the freeze-submissions script with different users and adding catches for the errors.  I have found that admin can edit the submission invitation, but does not have permission to edit the submission notes.  I need the change the submission note writers to ‘freeze’ the paper.  This makes me wonder if maybe this is a superuser function after all, not something we want to share with admins.  Getting paper info and creating review invitations I have no problem with.  But do we want admins to be able to edit a submission note?

Pam

**Pull request merged**